vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1016,6 +1016,11 @@ def maybe_setup_kv_connector():
                 if get_forward_context().attn_metadata is not None:
                     kv_connector.start_load_kv(get_forward_context())
 
+        def maybe_wait_for_save():
```

Comment:
note: in the `lm-cache` impl, this is what actually triggers the request to send